pkg/project/version.go

```diff
@@ -0,0 +1,33 @@
+package project
+
+const (
+	notAvailable = "n/a"
+)
+
+var (
+	description = "The release-operator manages chart configs for new releases."
+	gitSHA      = notAvailable
```

Comment:
We need to check if this works with `architect`. I am pretty sure it does not, because it says `-X main.gitCommit=%s`. I would also question if the variable thing works that way. I remember an issue in some other project and we had to use the plain string. I would prefer that. No need for the constant. See https://github.com/giantswarm/architect/blob/8910667e1e26464c1d6d4c8935f445afcdf03384/workflow/golang.go#L261.

Reply:
Yes we need to change the `architect` first. It should set all those variables (including old ones) for the time being.

Note it isn't a `const`. It's a `var`.

Reply:
`notAvailable` is a constant and I was questioning if the build flags even overwrite their content. I would simply go with a string.

Reply:
Ack. +1 on that.

Reply:
```suggestion
	gitSHA      = "n/a"
```

Reply:
I removed the constant, and will tackle architect afterward.